config/routes.rb

```diff
@@ -15,6 +15,7 @@
   get '/documents/:id/edit' => 'documents#edit', as: :edit_document
   patch '/documents/:id' => 'documents#update', as: :document
   get '/documents/:id' => 'documents#show'
+  post '/documents/:id/generate-path' => 'documents#generate_path'
```

Comment:
Can this be simplified to a `GET` now that we don't perform writes in the action?